config/locales/notices/en.yml

```diff
@@ -26,6 +26,9 @@ en:
         access government services. We use industry-standard monitoring and
         security practices to protect your data. %{link}
       link: Learn more ›
+    use_diff_email:
+      text: Or, %{link}
```

Comment:
question, how do you all feel about 'approving' PRs like this when they are generally good to go modulo a small comment / fix, so that I can hit the big green button once completed?